Bottom: f475b3cd799d8f58d2e290937a6b48e55be6e70d
Top:    f3ede7bfa41513e4b531da09a39e989bd9e6377d
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 17:32:12 -0400

Refresh of temp

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index a853727..56dac55 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -338,11 +338,6 @@ static int __devinit psc_ac97_of_probe(struct of_device *op,
 	/* Go */
 	out_8(&regs->command, MPC52xx_PSC_TX_ENABLE | MPC52xx_PSC_RX_ENABLE);
 
-	id1 = psc_ac97_read(&ac97, AC97_VENDOR_ID1);
-	id2 = psc_ac97_read(&ac97, AC97_VENDOR_ID2);
-
-	dev_info(&op->dev, "Codec ID is %04x %04x\n", id1, id2);
-
 	return 0;
 }
